src/model/core/fuel.jl

```diff
@@ -323,7 +323,7 @@ function fuel!(EP::Model, inputs::Dict, setup::Dict)
 end
 
 
-function resources_with_fuel(rs::Vector{AbstractResource}, fuel_name::AbstractString)
+function resources_with_fuel(rs::Vector{<:AbstractResource}, fuel_name::AbstractString)
```

Comment:
Are these changes to the types intentional?

Reply:
Yes !!! Very good question, @gmantegna . That change allows for the possibility of having just one type of resource (even though there could be multiple generators using the same resource), which is a possibility (and in fact, the DC-OPF example cases precisely has that). The previous way the code was written doesn't allow for this possibility.